src/SriServiceProvider.php

```diff
@@ -76,11 +76,11 @@ private function parseAndGenerateUrl(string $path, string $href, bool $crossOrig
 
     private function generateJsUrl(string $href, string $integrity): HtmlString
     {
-        return new HtmlString("<script src='{$href}' {$integrity}></script>");
+        return new HtmlString('<script src="{$href}" {$integrity}></script>');
```

Comment:
Careful that by doing this, the variables won't be parsed.

Single quotes can't parse variables the way double-quoted string can.